### PR TITLE
fix: trace table pagination

### PR DESCRIPTION
**Pull request by makeavish**

### CODE REVIEW

File: frontend/src/container/Trace/TraceTable/index.tsx [hunk 1]
```diff
@@ -129,7 +129,12 @@ function TraceTable(): JSX.Element {
 		if (!Array.isArray(sort)) {
 			const { order = spansAggregateOrder } = sort;
 			if (props.current && props.pageSize) {
-				const spanOrder = order === 'ascend' ? 'ascending' : 'descending';
```
Comment (palashgdev) [inline frontend/src/container/Trace/TraceTable/index.tsx:132]: make a util for this one don't write logic in the component itself

**Review by palashgdev — CHANGES_REQUESTED** [review]

File: frontend/src/container/Trace/TraceTable/index.tsx (hunk 1 above)
Comment (makeavish) [reply frontend/src/container/Trace/TraceTable/index.tsx:132]: Done

File: frontend/src/container/Trace/TraceTable/index.tsx [hunk 2]
```diff
@@ -129,7 +139,7 @@ function TraceTable(): JSX.Element {
...
+				const spanOrder = getOrder(order as string);
```
Comment (palashgdev) [inline frontend/src/container/Trace/TraceTable/index.tsx:142]: check if order is string

File: frontend/src/container/Trace/TraceTable/index.tsx [hunk 3]
```diff
@@ -121,6 +121,16 @@ function TraceTable(): JSX.Element {
 		return '';
 	};
 
+	const getOrder = (order: string): string => {
```
Comment (palashgdev) [inline frontend/src/container/Trace/TraceTable/index.tsx:124]: move this one in utils

**Review by palashgdev — CHANGES_REQUESTED** [review]

File: frontend/src/container/Trace/TraceTable/index.tsx (hunk 2 above)
Comment (makeavish) [reply frontend/src/container/Trace/TraceTable/index.tsx:142]: We are sending param as string.

**Review by palashgdev — APPROVED** [review]

**Pull request merged**